client/state/posts/test/actions.js

```diff
@@ -153,4 +163,70 @@ describe( 'actions', () => {
 			} ).catch( done );
 		} );
 	} );
+
+	describe( '#requestSitePost()', () => {
+		before( () => {
+			nock( 'https://public-api.wordpress.com:443' )
```

Comment:
>maybe we should look at fixing wpcom.js to make it easier to test through.

Yeah, I've used and seen a few different approaches to mocking WPCOM.js requests, though I'm mostly content with this one (see also: [mocking WPCOM.js methods gets hairy too](https://github.com/Automattic/wp-calypso/blob/b29d2b1c31642e8f34a1b7f2491401af5d35b62b/client/lib/media/test/actions.js#L56-L71)). 

Maybe some abstractions or constants to reduce the chance of changes breaking tests? I dunno, seems a bit premature, and this doesn't feel particularly troubling to me.